packages/apps/plugins/plugin-space/src/SpacePlugin.tsx

```diff
@@ -81,7 +82,7 @@ import {
 } from './types';
 import { SHARED, updateGraphWithSpace } from './util';
 
-const ACTIVE_NODE_BROADCAST_INTERVAL = 30_000;
+const ACTIVE_NODE_BROADCAST_INTERVAL = 30000;
```

Comment:
but… formatting!

Reply:
I'm thinking we should bring this interval down, especially now that I'm batching updates per space. CC @wittjosiah? Would love to make it 3 or 5 seconds.

Reply:
it was made 30s because we were trying to reduce the noise on the network at one point. Not sure if that's still an issue or not (cc @nferch )

Reply:
I'm happy to address this as another issue. In my video demo I was using 3000ms.